access/slack/bot.go

```diff
@@ -148,17 +177,30 @@ func (b *Bot) msgSections(reqID string, reqData RequestData, status string) []sl
 	if reqData.Roles != nil {
 		msgFieldToBuilder(&builder, "Role(s)", strings.Join(reqData.Roles, ","))
 	}
+	if reqData.RequestReason != "" {
+		msgFieldToBuilder(&builder, "Reason", reqData.RequestReason)
+	}
+	if b.webProxyURL != nil {
+		reqURL := *b.webProxyURL
+		reqURL.Path = lib.BuildURLPath("web", "requests", reqID)
+		msgFieldToBuilder(&builder, "Link", reqURL.String())
+	} else {
+		if status != "EXPIRED" {
+			msgFieldToBuilder(&builder, "Approve", fmt.Sprintf("tctl requests approve %s", reqID))
+			msgFieldToBuilder(&builder, "Deny", fmt.Sprintf("tctl requests deny %s", reqID))
```

Comment:
For teleport version`6.1` (coming out in ~3 weeks) onwards the recommended method of approval/denial via CLI will be:
- `tsh request review --aprove <request-id>`
- `tsh request review --deny <request-id>`

Probably best to just start recommending that now.